### PR TITLE
Feature/ability to create&view a page

### PR DESCRIPTION
**Pull request by AlexKapic**

### CODE REVIEW

File: frontend/src/components/toolbar/components/page-item/page-item.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import styles from '../../styles.module.scss';
+import { useAppDispatch } from 'hooks/hooks';
+import { pagesActions } from 'store/pages';
+import Button from 'react-bootstrap/Button';
+import { IPageRequest } from 'common/interfaces/pages';
+
+type Props = {
+  title: string | null;
+  id: string | null;
+  childrenPages: IPage[] | null;
```
Comment (ursarik) [inline frontend/src/components/toolbar/components/page-item/page-item.tsx:16]:
```suggestion
  title?: string;
  id?: string;
  childrenPages?: IPage[];
```

File: frontend/src/components/pages/components/page-content/page-content.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  const Content: React.FC = () => {
+    return (
+      <div className="content">
+        <div className="container-fluid p-0">
+          <h1 className="h3 mb-3">{pageTitle || 'New Page'}</h1>
+          <div className="row">
+            <div className="col-12">
+
+              <div className="card">
```
Comment (SirRedrick) [inline frontend/src/components/pages/components/page-content/page-content.tsx:19]: react-bootstrap has a card component

File: shared/package.json [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
-    "lint": "npm run lint:js",
-    "build:ts": "tsc && tscpaths -p tsconfig.json --src src --out build",
-    "build": "npm run build:ts && cp package.json build"
-  },
-  "devDependencies": {
-    "tscpaths": "0.0.9",
-    "typescript": "4.3.5"
-  },
-  "dependencies": {}
+    "name": "shared",
```
Comment (SirRedrick) [inline shared/package.json:2]: Remove unnecessary  indent

**Review by SirRedrick — COMMENTED** [review]
Remove `.gitkeep` files from folders that are not empty, they are there only so a folder is pushed to the repo

**Review by aleksandr-kuharenko — APPROVED** [review]

**Review by ursarik — APPROVED** [review]

**Pull request merged**